### PR TITLE
Support More localization

### PR DESCRIPTION
**Pull request by SinghRajenM**
Support localization for few more items:
1. All the controls on "Tab rename" dialog
2. Label ("Max Size: ") on "recent file history" under preference dialog
3. Label ("Tab Size: ") on "Language" under preference dialog

## Edit
4. Label ("Nb of column: ") on "editing" under preference dialog
5. Label ("Nb char: ") on "auto-completion" under preference dialog
6. Title and static control on the dialog which appears when "Create New...", "Rename" and "Save as" is clicked.

### CODE REVIEW

File: PowerEditor/installer/nativeLang/english.xml [hunk 1]
```diff
@@ -1196,6 +1196,12 @@ Do you want to launch Notepad++ in Administrator mode?"/>
             <finder-select-all value="Select all"/>
             <finder-clear-all value="Clear all"/>
             <finder-open-all value="Open all"/>
+            <common-ok value="OK"/>
+            <common-cancel value="Cancel"/>
+            <tabrename-title value="Rename Current Tab"/>
+            <tabrename-newname value="New Name: "/>
+            <recent-file-histroy-maxfile value="Max File: "/>
```
Comment (MAPJe71) [inline PowerEditor/installer/nativeLang/english.xml]: Typo: "...-hist**ro**y-..." --> "...-hist**or**y-...".

File: PowerEditor/installer/nativeLang/hindi.xml [hunk 2]
```diff
@@ -1174,6 +1174,12 @@
 		<finder-select-all value="सब सेलेक्ट करें"/>
 		<finder-clear-all value="सब क्लियर करें"/>
 		<finder-open-all value="सब ओपन करें"/>
+		<common-ok value="ओके"/>
+		<common-cancel value="रद्द करें"/>
+		<tabrename-title value="वर्तमान टैब का नाम बदलें"/>
+		<tabrename-newname value="नया नाम: "/>
+		<recent-file-histroy-maxfile value="फाइल सीमा:"/>
```
Comment (MAPJe71) [inline PowerEditor/installer/nativeLang/hindi.xml]: Typo: "...-hist**ro**y-..." --> "...-hist**or**y-...".

File: PowerEditor/src/WinControls/Preference/preferenceDlg.cpp [hunk 3]
```diff
@@ -1474,8 +1475,9 @@ INT_PTR CALLBACK RecentFilesHistoryDlg::run_dlgProc(UINT message, WPARAM wParam,
 
 				case IDC_MAXNBFILEVAL_STATIC:
 				{
+					generic_string staticText = pNativeSpeaker->getLocalizedStrFromID("recent-file-histroy-maxfile", TEXT("Max File: "));
```
Comment (MAPJe71) [inline PowerEditor/src/WinControls/Preference/preferenceDlg.cpp]: Typo: "...-hist**ro**y-..." --> "...-hist**or**y-...".

**Comment by rddim** [comment]
@SinghRajenM could you add please support for:

1. Label ("Nb of column: ") on "editing" under preference dialog
2. Label ("Nb char: ") on "auto-completion" under preference dialog

**Comment by SinghRajenM** [comment]
@rddim and @MAPJe71 Done!!!

File: PowerEditor/installer/nativeLang/english.xml [hunk 4]
```diff
@@ -1196,6 +1196,18 @@ Do you want to launch Notepad++ in Administrator mode?"/>
             <finder-select-all value="Select all"/>
             <finder-clear-all value="Clear all"/>
             <finder-open-all value="Open all"/>
+            <common-ok value="OK"/>
+            <common-cancel value="Cancel"/>
```
Comment (donho) [inline PowerEditor/installer/nativeLang/english.xml]: `OK` and `Cancel` shouldn't be localized since it's the system takes charge in them.

Comment (donho) [reply PowerEditor/installer/nativeLang/english.xml]: Hmm... never mind, I was wrong.

Comment (donho) [reply PowerEditor/installer/nativeLang/english.xml]: Yes, it's pretty good common-ok & common-cancel.

**Comment by donho** [comment]
@SinghRajenM  Could you solve the conflict please?

File: PowerEditor/installer/nativeLang/english.xml (hunk 4 above)
Comment (SinghRajenM) [reply PowerEditor/installer/nativeLang/english.xml]: > Hmm... never mind, I was wrong.

No Problem. I have resolved conflict now.

**Comment by SinghRajenM** [comment]
@donho I have resolved the conflict.

**Comment by donho** [comment]
Thank you @SinghRajenM

**Pull request closed without merging**